executor/write_test.go

```diff
@@ -642,6 +642,12 @@ func (s *testSuite) TestLoadData(c *C) {
 	r := tk.MustQuery(selectSQL)
 	r.Check(nil)
 
+	sc := ctx.GetSessionVars().StmtCtx
+	originIgnoreTruncate := sc.IgnoreTruncate
+	defer func() {
```

Comment:
If you pursue a good habit, you should reset context behind line 675. If you want a test in this function, you may use the dirty var.